_pages/how-to-submit.md

```diff
@@ -0,0 +1,213 @@
+* **Citation management and bibliometrics:** these are packages that facilitate managing references.
+* **Scientific software wrappers:** these tools provide a Python interface for existing scientific packages written in other languages.
+* **Database interoperability:** these packages deal with bindings and wrappers for database APIs.
+
+Similar to the previous section, you can indicate which categories are applicable to your package in one of two ways:
+* adding an "X" (without quotation marks) inside the brackets in the Markdown version of the issue, so that it reads like this: [X]
+* submitting the issue without making any changes to this section, and then clicking on the checkboxes when they're rendered in the submitted issue.
+
+You can read more about each of these categories in the [Package Scope](https://www.pyopensci.org/software-peer-review/index.html) section of the [pyOpenSci Peer Review Guide](https://www.pyopensci.org/software-peer-review/about/package-scope.html).
+
+### Domain specific
+
+This section is currently evolving, and it's OK if your package doesn't fit into a specific listed domain.
+
+### Community Partnerships
+
+Similar to the Domain specific section, our Community Partnerships are continuously growing and changing, and it's OK if your package isn't affiliated with one of the listed partners.
+
+### For all submissions
+
+This section is important, as it helps us more holistically evaluate your package and determine whether or not it's a good fit for pyOpenSci review. Each of the following sections should be addressed/answered:
+* *Explain how and why the package falls under the categories you indicated above.* Here's where you'll provide us with information that connects your package and its functionality to both its domain and technical capabilities.
+* *Who is the target audience and what are scientific applications of this package?* This helps us understand who will be using the package.
+* *Are there other Python packages that accomplish the same thing? If so, how does yours differ?* This helps us get a handle on other packages that fall within the same domain, as well as ensure that efforts aren't being duplicated.
+* *If you made a pre-submission enquiry, please paste the link to the corresponding issue, forum post, or other discussion, or `@tag` the editor you contacted.* This is an important section, as it links your pre-submission inquiry with your submission.
+
+### Technical checks
+
+It's important that your package meet each of the qualifications in this section. If you are missing one or more of these items, you can learn more about how to implement them in our [pyOpenSci Packaging Guide](https://www.pyopensci.org/python-package-guide/). Taken as a whole, these components represent best practices in Python packaging, and as such its important that they be present in all pyOpenSci-approved packages.
```

Comment:
```suggestion
This section contains a list of community-developed core elements that all pyOpenSci Python packages should have such as documentation, tests and continuous integration setup. These package elements ensure that your package is both easier to maintain, easier to contribute to and documented well enough that a new user can get started using your package without too much additional guidance. 

It's important that your package meet each of the qualifications in this section. If you are missing one or more of these items, you can learn more about how to implement them in our [pyOpenSci Packaging Guide](https://www.pyopensci.org/python-package-guide/) or you can ask a question on our Discourse forum. 
```

i think in general it would be good for sections that aren't clear to at the top - 
1. define briefly in a few sentences why that section exists in our template and then 
3. lower down, explain any elements of it that may be unclear. 

...Being sure to try to align with our mission and values where ever you can. And also try to emphasize that tehse things aren't random, they are community developed. 

What i've rewritten here tries to do that for the Techincal checks section.